config/jobs/kubernetes/cloud-provider-ibmcloud/cloud-provider-ibmcloud-periodics.yaml

```diff
@@ -150,3 +150,81 @@ periodics:
                   echo "ERROR: Serial k8s Conformance tests exited with code: $rc2"
                   exit $rc2
               fi
+  - name: ci-kubernetes-ppc64le-e2e-node-latest-kubetest2
+    interval: 6h
+    cluster: k8s-infra-ppc64le-prow-build
+    labels:
+      preset-ibmcloud-cred: "true"
+    decorate: true
+    decoration_config:
+      timeout: 220m
+    extra_refs:
+      - base_ref: main
+        org: kubernetes-sigs
+        repo: provider-ibmcloud-test-infra
+        workdir: true
+    annotations:
+      description: Runs e2e-node tests using kubetest2 on IBM powervs
+      testgrid-dashboards: ibm-k8s-e2e-node-ppc64le, sig-node-ppc64le
+      testgrid-tab-name: ci-kubernetes-ppc64le-e2e-node-latest-kubetest2
+    spec:
+      containers:
+        - image: us-central1-docker.pkg.dev/k8s-staging-test-infra/images/kubekins-e2e:v20250227-3a13bdd784-master
+          env:
+            - name: "BOSKOS_HOST"
+              value: "boskos.test-pods.svc.cluster.local"
+            - name: "USER"
+              value: "ci-kubernetes-ppc64le-e2e-node-latest-kubetest2"
+          resources:
+            requests:
+              cpu: 8
```

Comment:
any reason for assigning `8` vcpus? this really means - grabbing an entire worker node, it will be great if we you can reduce the requests/limits for cpu!

Reply:
I startedoff looking at the jobs in https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/ec2-containerd.yaml where all of had Garanteed 8 vcpus.
Yes you are right, a lesser cpu value should be enough in our case!
Our existing e2e-node job that has been running internally, used only 1500m vcpus https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/periodic/kubernetes/test-kubernetes-periodics.yaml#L24

I shall change this job to use 2 vcpus.